src/main/java/entities/Timeslot.java

```diff
@@ -100,6 +100,10 @@ public boolean equals(Object o) {
 
     @Override
     public int compareTo(Timeslot that) {
-        return this.start.compareTo(that.getStart());
+        int dayCompare = this.day.compareTo(that.getDay());
+        if (dayCompare == 0) {
+            return this.start.compareTo(that.getStart());
```

Comment:
this seems unnecessary, especially when DateTime implements compare already

Reply:
fine for now, should modify for phase 2.